pivot/static/pivot/js/main.js

```diff
@@ -550,10 +551,6 @@ function populateCollegeDropdown() {
         }
         $("#dropdownMenu").attr("data-campus", $(this).attr("class"));
         toggleGo();
-        if (window.location.href.indexOf("course-gpa") > -1) {
-            // creates a user like click
-            setTimeout(prepareResults, 10);
-        }
```

Comment:
Looking back through a git blame, it looks like this behavior was intentional, but might need to be consistent accross the different views.